src/server.c

```diff
@@ -2911,46 +2911,64 @@ void resetErrorTableStats(void) {
 
 /* ========================== Redis OP Array API ============================ */
 
-void redisOpArrayInit(redisOpArray *oa) {
-    oa->ops = NULL;
+void redisOpArrayReset(redisOpArray *oa) {
```

Comment:
i think this function is unnecessary (only used in one place, we can just copy the two lines there)

Reply:
capacity, numops: stays the same
len -> used

please add comment in the struct